src/main/java/our/yurivongella/instagramclone/service/CommentService.java

```diff
@@ -0,0 +1,141 @@
+import our.yurivongella.instagramclone.domain.comment.Comment;
+import our.yurivongella.instagramclone.domain.comment.CommentLike;
+import our.yurivongella.instagramclone.domain.comment.CommentLikeRepository;
+import our.yurivongella.instagramclone.domain.comment.CommentRepository;
+import our.yurivongella.instagramclone.domain.member.Member;
+import our.yurivongella.instagramclone.domain.member.MemberRepository;
+import our.yurivongella.instagramclone.domain.post.Post;
+import our.yurivongella.instagramclone.domain.post.PostRepository;
+import our.yurivongella.instagramclone.util.SecurityUtil;
+
+@Transactional(readOnly = true)
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class CommentService {
+    private final MemberRepository memberRepository;
+    private final PostRepository postRepository;
+    private final CommentRepository commentRepository;
+    private final CommentLikeRepository commentLikeRepository;
+
+    public List<CommentResponseDto> getComments(Long postId) {
+        Long currentMemberId = SecurityUtil.getCurrentMemberId();
+        Member member = memberRepository.findById(currentMemberId)
+                                        .orElseThrow(() -> new NoSuchElementException("유저가 존재하지 않습니다."));
+
+        return postRepository.findById(postId).orElseThrow(() -> new NoSuchElementException("해당 게시물이 없습니다."))
+                             .getComments()
+                             .stream()
+                             .map(c -> CommentResponseDto.of(c, member))
+                             .collect(Collectors.toList());
+    }
+
+    @Transactional
+    public CommentResponseDto createComment(Long postId, CommentCreateDto commentCreateDto) {
+        Long currentMemberId = SecurityUtil.getCurrentMemberId();
+        Member member = memberRepository.findById(currentMemberId)
+                                        .orElseThrow(() -> new UsernameNotFoundException("유저가 존재하지 않습니다."));
+
+        Post post = postRepository.findById(postId)
+                                  .orElseThrow(() -> new NoSuchElementException("해당 게시물이 없습니다."));
+
+        Comment comment = new Comment();
+        comment.create(member, post, commentCreateDto.getContent());
+        commentRepository.save(comment);
+        return CommentResponseDto.of(comment, member);
+    }
+
+    @Transactional
+    public ProcessStatus deleteComment(Long commentId) throws Exception {
+        Long currentMemberId = SecurityUtil.getCurrentMemberId();
+        Member member = memberRepository.findById(currentMemberId)
+                                        .orElseThrow(() -> new UsernameNotFoundException("유저가 존재하지 않습니다."));
+
+        Comment comment = commentRepository.findById(commentId)
+                                           .orElseThrow(() -> new NotFoundException("존재하지 않는 댓글입니다."));
+
+        if (!member.getId().equals(comment.getMember().getId())) {
+            log.error("유저가 일치하지 않습니다.");
+            return ProcessStatus.FAIL;
+        }
```

Comment:
`Member` 클래스에 다른 멤버와 비교하는 `equals` 메소드가 만들어져 있습니다.

```suggestion
        if (!currentMember.equals(targetMember)) {
            log.error("유저가 일치하지 않습니다.");
            return ProcessStatus.FAIL;
        }
```

Reply:
또는 가독성 있게 `notEquals` 메소드를 만들어서 사용해도 괜찮을 것 같아요

```java
public boolean notEquals(Member other) {
    return !equals(other);
}
```

Reply:
Done -> [8e2aad2](https://github.com/INSPJT/instagram_clone_backend/pull/33/commits/8e2aad2aac2f82d4abe29baaa1d778a135de2178)